compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -353,15 +353,15 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<v
 private:
   // Check if
   // 1. node is const
-  // 2. node was not quantized
+  // 2. node's dtype is float32
   bool is_quantizable(loco::Node *node)
   {
     auto const_node = dynamic_cast<luci::CircleConst *>(node);
     if (not const_node)
       return false;
 
-    // Skip if this is already quantized
-    if (is_quantized(const_node))
```

Comment:
```cpp
bool is_quantized(const CircleNode *node)
{
  return node->quantparam() != nullptr &&
         (node->dtype() == loco::DataType::U8 ||  // activation, weight (uint8 quant)
          node->dtype() == loco::DataType::S16 || // activation, weight (int16 quant)
          node->dtype() == loco::DataType::S32 || // bias (uint8 quant)
          node->dtype() == loco::DataType::S64);  // bias (int16 quant)
}
```

Note that `is_quantized` checks if quantparam exists. quantparam is removed before QuantizeDequantizeWeigthsPass is invoked, so `is_quantized` always returns false.

This PR fixes this by not checking the existence of quantparam.